### PR TITLE
fix unresponsive table for children and complaints

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### What tests were added (if relevant), and do all tests (rspec, karma, and integration CALS tests) pass?
- n/a

#### Which User story does this relate to?
-#146491781

#### Brief feature description
- fix the children and complaints table for less than 600px

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
- no

#### Code coverage percent diff as a result of this PR?
- 100

#### JS errors diff as a result of this PR?
- 0

### CODE REVIEW

**Comment by ratneshraval** [comment]
/public files should not be committed. Can you remove them?

File: app/javascript/facility/stylesheets/facility.scss [hunk 1]
```diff
@@ -107,6 +107,46 @@
     color: #2b2b2b;
 }
 
+@media screen and (max-width: 600px){
```
Comment (anuroop15) [inline app/javascript/facility/stylesheets/facility.scss]: @keith, are you targeting mobile screens? As we are following Bootstrap guidelines, mobile screens should be 767px and below. Is there aa . reason you target upto 600 only?

Comment (keith) [inline app/javascript/facility/stylesheets/facility.scss]: Wrong Keith 😄

Comment (lauherkcambria) [inline app/javascript/facility/stylesheets/facility.scss]: apologies @keith! 🙇 🙏

Comment (lauherkcambria) [inline app/javascript/facility/stylesheets/facility.scss]: @anuroop15, i set it as 600 as I thought that was the standard -- can be corrected to 767, i was going off of the standard: 
`@media (min-width:600px) and (max-width:767px) {}`

which is probably why. i'll update and please approve. thanks

**Pull request merged**